components/PredictPage/Activity.jsx

```diff
@@ -0,0 +1,192 @@
+import { getPredictionDaa, getPredictionTxs } from 'common-util/api/dune';
+import { PREDICTION_ECONOMY_DASHBOARD_URL } from 'common-util/constants';
+import SectionWrapper from 'components/Layout/SectionWrapper';
+import { Card } from 'components/ui/card';
+import { Popover } from 'components/ui/popover';
+import { ExternalLink } from 'components/ui/typography';
+import Image from 'next/image';
+import { useMemo } from 'react';
+import useSWR from 'swr';
+
+const fetchMetrics = async () => {
+  const [dailyActiveAgents, transactions] = await Promise.allSettled([
+    getPredictionDaa(),
+    getPredictionTxs(),
+  ]);
+
+  return {
+    dailyActiveAgents:
+      dailyActiveAgents.status === 'fulfilled' ? dailyActiveAgents.value : null,
+    traderTxs:
+      transactions.status === 'fulfilled' ? transactions.value.traderTxs : null,
+    mechTxs:
+      transactions.status === 'fulfilled' ? transactions.value.mechTxs : null,
+    marketCreatorTxs:
+      transactions.status === 'fulfilled'
+        ? transactions.value.marketCreatorTxs
+        : null,
+    totalTxs:
+      transactions.status === 'fulfilled' ? transactions.value.totalTxs : null,
+  };
+};
+
+const usePersistentSWR = (key, fetcher) =>
+  useSWR(key, fetcher, {
+    revalidateIfStale: false,
+    revalidateOnFocus: false,
+  });
+
+export const Activity = () => {
+  const { data: metrics } = usePersistentSWR(
+    'predictionActivityMetrics',
+    fetchMetrics,
+  );
+
+  const data = useMemo(
+    () => [
+      {
+        id: 'traders',
+        label: (
+          <div className="flex flex-col gap-2 mb-3">
+            <Image
+              alt="Predict"
+              src="/images/predict-page/traders.png"
+              width="72"
+              height="32"
+            />
+            <span className="text-base font-semibold text-black">Traders</span>
+          </div>
+        ),
+        subText: 'transactions',
+        value: metrics?.traderTxs?.toLocaleString(),
```

Comment:
can ignore if you want -- `metrics?.traderTxs?.toLocaleString()` is used a few times can probably make it const

Reply:
those are different, e.g. traderTxs, mechTxs, etc

Reply:
[okay](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExMmR5ZnQ0cGVtYmVnbGllbHdtcThrOWg1eTlraTFmMmY3anZ5dnl6ZCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/3ov9k00Z2q2BLWf0Lm/giphy.gif)